scripts/aggregate-translations/aggregate-translations-cli.js

```diff
@@ -25,13 +25,15 @@ commander
   .option('-d, --directories [directories]', 'Regex pattern to glob search for translations', addCustomDirectory)
   .option('-l, --locales <locales>', 'The list of locale codes aggregate on and combine into a single, respective translation file ', localeList, supportedLocales)
   .option('-o, --outputDir [outputDir]', 'The output location of the generated configuration file', './aggregated-translations')
+  .option('-c, --config [baseDir]', 'The path to pull terra i18n config from')
```

Comment:
'-c, --config ~[baseDir]~ **[configPath]**',

Reply:
This was all copied from you. 😜

Reply:
derp, no it wasn't